### ISSUE TITLE
System.NotImplementedException: psm

### ISSUE THREAD

**Issue by NoahRoseLedesma** [Ryujinx/Ryujinx#465]
OSX 10.12.1

I get the following error when I attempt to run the LovePotion game engine nro.

Update:
NRO works fine on switch 

```
Unhandled Exception: System.NotImplementedException: psm
   at Ryujinx.HLE.HOS.Services.ServiceFactory.MakeService(Horizon System, String Name) in C:\projects\ryujinx\Ryujinx.HLE\HOS\Services\ServiceFactory.cs:line 189
   at Ryujinx.HLE.HOS.Services.Sm.IUserInterface.GetService(ServiceCtx Context) in C:\projects\ryujinx\Ryujinx.HLE\HOS\Services\Sm\IUserInterface.cs:line 70
   at Ryujinx.HLE.HOS.Services.IpcService.CallMethod(ServiceCtx Context) in C:\projects\ryujinx\Ryujinx.HLE\HOS\Services\IpcService.cs:line 96
   at Ryujinx.HLE.HOS.Ipc.IpcHandler.IpcCall(Switch Ns, Process Process, AMemory Memory, KSession Session, IpcMessage Request, Int64 CmdPtr) in C:\projects\ryujinx\Ryujinx.HLE\HOS\Ipc\IpcHandler.cs:line 33
   at Ryujinx.HLE.HOS.Kernel.SvcHandler.ProcessIpcRequest(Object State) in C:\projects\ryujinx\Ryujinx.HLE\HOS\Kernel\SvcSystem.cs:line 231
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.ThreadPoolWorkQueue.Dispatch()
Abort trap: 6
```

**Comment by AcK77** [comment]
Could you provide the NRO you tried to load please? Thanks!

**Comment by NoahRoseLedesma** [comment]
Sure. [Download link](https://filebin.net/fo84g43saa0qnk1f)

**Comment by gdkchan** [comment]
Should be fixed with #467, however I still need to test your sample.

**Comment by kennyvv** [comment]
I can confirm that LovePotion works again.

**Comment by AcK77** [comment]
Fixed in #467 and #485. LovePotion running fine.

![unknown](https://user-images.githubusercontent.com/4905390/47873587-eb66e700-de11-11e8-9788-7d942d3ac46b.png)
